public/app/core/components/AppChrome/DockedMegaMenu/MegaMenuItem.tsx

```diff
@@ -46,11 +46,10 @@ export function MegaMenuItem({ link, activeItem, level = 0, onClick }: Props) {
           <div
             className={cx(styles.labelWrapper, {
               [styles.isActive]: isActive,
-              [styles.hasActiveChild]: hasActiveChild,
             })}
           >
             <FeatureHighlightWrapper>
-              <div className={styles.iconWrapper}>{level === 0 && <MegaMenuItemIcon link={link} />}</div>
+              <div className={styles.iconWrapper}>{level === 0 && <Icon name={iconName ?? 'link'} size="xl" />}</div>
```

Comment:
maybe

```suggestion
              <div className={styles.iconWrapper}>{level === 0 && link.icon && <Icon name={toIconName(link.icon) ?? 'link'} size="xl" />}</div>
```

then you don't need the ternary on line 32